openassessment/xblock/ui_mixins/mfe/test_mfe_mixin.py

```diff
@@ -0,0 +1,160 @@
+"""
+Tests for XBlock handlers for the ORA MFE BFF
+"""
+from contextlib import contextmanager
+import json
+from unittest.mock import patch
+from submissions import api as submission_api
+from openassessment.xblock.test.base import XBlockHandlerTestCase, scenario
+from openassessment.xblock.ui_mixins.mfe.constants import ErrorCodes
+from openassessment.xblock.apis.submissions.errors import (
+    SubmissionValidationException,
+    DraftSaveException,
+    StudioPreviewException,
+    MultipleSubmissionsException,
+    AnswerTooLongException,
+    EmptySubmissionError,
+    SubmitInternalError
+)
+
+
+def assert_error_response(response, status_code, error_code, context=''):
+    assert response.status_code == status_code
+    assert response.json['error'] == {
+        'error_code': error_code,
+        'error_context': context
+    }
+
+
+def assert_called_once_with_helper(mock, expected_first_arg, expected_additional_args):
```

Comment:
Nit: could be named better, minimally. Bonus points for the cleanliness of having one "expected_args" item instead of split "first" and "additional" args.

```suggestion
def assert_called_once_with_args(mock, expected_args):
```

Reply:
I should add a docstring and change the name of additional_args. I don't really want to do a check of all args, I just want to check that the first arg is a specific value and there are the expected number of additional args. I don't see any value in checking the equality of data api wrapper objects, and I think we're constructing different wrapper objects anyway, making a shallow equality test meaningless